Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -521,21 +522,23 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor,
   void LoadTileset();
   void DestroyTileset();
   void MarkTilesetDirty();
-  Cesium3DTiles::ViewState CreateViewStateFromViewParameters(
-      const FVector2D& viewportSize,
-      const FVector& location,
-      const FRotator& rotation,
-      double fieldOfViewDegrees) const;
 
   struct UnrealCameraParameters {
+    bool isPerspective;
+    bool isOrthographic;
```

Comment:
Can `isPerspective` and `isOrthographic` ever have the same value? The frustum type is derived from that with 
```
  if (camera.isOrthographic) {
    frustumType = Cesium3DTiles::ViewState::FrustumType::Orthographic;
  } else {
    frustumType = Cesium3DTiles::ViewState::FrustumType::Perspective;
  }
```
and the method where they are obtained is implemented as
```
bool FEditorViewportClient::IsOrtho() const
{
	return !IsPerspective();
}
```
so I think these fields might be redundant...